src/classes/project_data.py

```diff
@@ -60,15 +60,15 @@ def __init__(self):
         self.new()
 
     def needs_save(self):
-        """Returns if project data Has unsaved changes"""
+        """Returns if project data has unsaved changes"""
         return self.has_unsaved_changes
 
     def get(self, key):
-        """ Get copied value of a given key in data store """
+        """Get copied value of a given key in data store"""
 
         # Verify key is valid type
         if not key:
-            log.warning("Cannot get empty key.")
+            log.debug("ProjectDataStore cannot get empty key (%s)", key)
```

Comment:
There's not a lot of point in logging `key` here — we know it's empty.

Also, I would _not_ lower this one to debug, personally — that code path that should never be reached except through bad user data or programmer error, so it's one of those "could signify an error resulting in potential user data loss" ones. Anytime the code tries to query an empty key, we want to shout about it.

Reply:
(If I mistakenly gave a different impression in #3708, my apologies.)

Reply:
I undid this, sorry for my missunderstanding.

As a sidenote: `boo(key)` could (without knowing the application) mean many things:

* key = ""
* key = None
* key = []
* key = False
* key = 0